File: JVM/Kotlin/empty_java/app/src/main/java/com/example/empty_java/MainActivity.java
package com.example.empty_java;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);


        Button btn1 = findViewById(R.id.button1);

        btn1.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {

                Toast tot = Toast.makeText(MainActivity.this, "我爱我的莲儿", Toast.LENGTH_LONG);
                tot.show();

            }
        });


    }
}

File: JVM/Kotlin/empty/app/src/main/java/com/example/empty/MainActivity.kt
package com.example.empty

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.widget.Toast
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        /**
         * 按钮1的单击事件
         */
        button1.setOnClickListener {
            Toast.makeText(this, "点击了", Toast.LENGTH_SHORT).show()
        }

        /**
         * 按钮2的单击事件
         */
        button_password.setOnClickListener {

            Toast.makeText(this, editText_password.text, Toast.LENGTH_SHORT).show()

        }
    }
}
